Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/speaker/SpeakerAdapterItem.kt

```diff
@@ -21,19 +21,17 @@ class SpeakerAdapterItem internal constructor(val itemData: EventSpeaker) :
         AbstractFlexibleItem<SpeakerAdapterItem.ViewHolder>() {
 
     override fun bindViewHolder(adapter: FlexibleAdapter<IFlexible<RecyclerView.ViewHolder>>?,
-                                holder: ViewHolder, position: Int, payloads: MutableList<Any>?) {
+                                holder: ViewHolder,
+                                position: Int,
+                                payloads: MutableList<Any>?) {
 
         holder.name.text = itemData.name
         holder.bio.text = itemData.bio.getHtmlFormattedSpanned()
 
         val context = holder.name.context
 
-        Glide.with(context)
-                .load(itemData.pictureUrl)
-                .transform(CircleTransform(context))
-                .placeholder(R.drawable.emo_im_cool)
-                .crossFade()
-                .into(holder.avatar)
+        Glide.with(context).load(itemData.pictureUrl).transform(CircleTransform(context))
+                .placeholder(R.drawable.emo_im_cool).crossFade().into(holder.avatar)
```

Comment:
This chain here can stay one method per line for readability, but should be lined up with the `.with()`.

Reply:
I guess we stick to official kotlin style guide.

Added that and reformatted the code.